documentation/README.md

```diff
@@ -0,0 +1,25 @@
+# Adobe CIF cloud API
+
+## REST
+
+The Adobe CIF Cloud API follows the [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) principles which involve separating the APIs into logical resources that are manipulated using HTTP requests where the method (GET, POST, PUT, PATCH, DELETE) has specific meaning. As a consequence, the API can be consumed by all standard HTTP clients.
+
+## Authorization
+
+TBD
+
+## JSON
+
+All response payloads are in JSON format.
```

Comment:
Is this really correct? I think that this is more runtime/OW features so maybe we should just have link to that documentation.

Reply:
I think this is correct for CIF at the moment we only support JSON format. OW supports more yes but this is CIF doc.